test/EcdsaXi2023Cryptosuite.spec.js

```diff
@@ -30,6 +30,35 @@ const extraInformation = new Uint8Array([
 ]);
 const ecdsaXi2023Cryptosuite = createCryptosuite({extraInformation});
 
+const testSignedCredential = {
+  '@context': [
+    'https://www.w3.org/2018/credentials/v1',
+    {
+      AlumniCredential: 'https://example.org/examples#AlumniCredential',
+      alumniOf: 'https://schema.org/alumniOf'
+    },
+    'https://w3id.org/security/data-integrity/v2'
+  ],
+  id: 'https://example.edu/credentials/1872',
+  type: [ 'VerifiableCredential', 'AlumniCredential' ],
+  issuer: 'https://example.edu/issuers/565049',
+  issuanceDate: '2010-01-01T19:23:24Z',
+  credentialSubject: {
+    id: 'https://example.edu/students/alice',
+    alumniOf: 'Example University'
+  },
+  proof: {
+    type: 'DataIntegrityProof',
+    created: '2023-03-01T21:29:24Z',
+    verificationMethod: 'https://example.edu/issuers/' +
+    '565049#zDnaekGZTbQBerwcehBSXLqAg6s55hVEBms1zFy89VHXtJSa9',
+    cryptosuite: 'ecdsa-xi-2023',
+    proofPurpose: 'assertionMethod',
+    proofValue: 'z2Q8gjsP1zrCA8wLb6pErmGJic5Uw7723JAjXSTkdhib2' +
+    'xoH4LBR49LnbijfhjVMm7NfKYYiQxF2tC6pgEmqq9MtZ'
```

Comment:
Should be indented 2 spaces ... I'd do a suggestion here but github won't let me for some reason.